pkg/config/validator.go

```diff
@@ -280,8 +280,17 @@ func checkNodeHostname(node Node, idx int, result *Errors) *Errors {
 
 func checkNodeLabels(node Node, idx int, result *Errors) *Errors {
 	if node.NodeLabels != nil {
+		// Skip labels that include templates as these will not be valid
```

Comment:
Similar logic may be needed for annotations... not sure. The example works just fine, but there may be some case where templates can fail, but the rendered value would not.

Reply:
You can do this for annotations too now?

Reply:
Yep I added support for both. I can pull this part out if you'd like. Annotations can be a useful alternative because there is a short limit (63 characters) for label values.